requirements.txt

```diff
@@ -1 +1,2 @@
--e .
\ No newline at end of file
+-e .
+typing_extensions
```

Comment:
This is unfortunately the wrong location for requirements changes. Please place them in setup.cfg